VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -531,37 +522,27 @@ protected void hideSoftKeypad() {
     }
 
     public boolean showErrorMessage(String header, String message) {
-        return showErrorMessage(header, message, true);
-    }
+        hideErrorMessage();
+        alertDialog = new AlertDialog.Builder(this)
+                .setTitle(header)
+                .setMessage(message)
+                .setCancelable(false)
```

Comment:
Ok. I’ll try that.

On Tue, Aug 9, 2016 at 10:58 AM, Brian @ edX <notifications@github.com>
wrote:

> In VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java
> <https://github.com/edx/edx-app-android/pull/794#discussion_r74075517>:
>
> > @@ -531,37 +522,27 @@ protected void hideSoftKeypad() {
> >      }
> >
> >      public boolean showErrorMessage(String header, String message) {
> > -        return showErrorMessage(header, message, true);
> > -    }
> > +        hideErrorMessage();
> > +        alertDialog = new AlertDialog.Builder(this)
> > +                .setTitle(header)
> > +                .setMessage(message)
> > +                .setCancelable(false)
>
> Still don't think we should break the back button, how about using
> setCanceledOnTouchOutside(false) instead?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/edx/edx-app-android/pull/794/files/8fdfe5f3a21cec9d0bc9d4212121f0eb75e7b282#r74075517>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AR10T46OYsaPIGDRe15ASEmyvvNENJJPks5qeJWwgaJpZM4JfQbx>
> .
>



-- 
Regards,

Marc DiNino
Android Engineer
EDX